code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -294,6 +295,10 @@
 	if(nutrition < max_nutrition / 3 && isturf(loc))	//If we're hungry enough (and not being held/in a bag), we'll check our tile for food.
 		handle_eating()
 
+//Overriden by cat.
+/mob/living/simple_animal/proc/handle_radiobioluminescence()
+	return
```

Comment:
You don't have to put this on mob/living/simple_animal, you can just put it on the cat and do this:
```
/mob/living/simple_animal/cat/Life()
	. = ..()
	handle_radiation_light()
```
Also these procs and variable names are a mouthful. Please simplify them.